e2e/helpers/userConfig.ts

```diff
@@ -11,7 +11,7 @@ export type UserPrefixes = (typeof userPrefixes)[number]
 const generateUserName = (prefix: string) => {
   // uncomment to use static username for troubleshooting:
   // ...and ensure that usernames are unique per worker
-  // return `${prefix}-${process.env.TEST_PARALLEL_INDEX}`
+  // return `local-${prefix}-${process.env.TEST_PARALLEL_INDEX}`
```

Comment:
In local tests using static usernames, the lingering test users on the backend dev stack caused multiple users to appear in the user suggestion dropdown, when we expect one user to appear (e.g. `swc-e2e-user-validated-0` and `swc-e2e-user-validated-0d930519-5c31-4137-9def-8fe9ea483f22`). Add a prefix for local testing, so that the local static test users are easily selected.